### PR TITLE
Fixes typos cross_validation.rst

### PR DESCRIPTION
**Pull request by efekhari27**

### CODE REVIEW

File: python/doc/theory/meta_modeling/cross_validation.rst [hunk 1]
```diff
@@ -36,107 +37,109 @@ estimate is obtained by averaging over the splits.
...
+  :label: 4.3:7
 
-      \Delta^{(i)} \, = \,
-         \frac{h(\underline{x}^{(i)}) - \widehat{h}(\underline{x}^{(i)})}{1 - h_i}
+    \Delta^{(i)} \, = \,
+       \frac{h(\underline{x}^{(i)}) - \widehat{h}(\underline{x}^{(i)})}{1 - h_i}
 
-  where :math:`h_i` is the :math:`i`-th diagonal term of the matrix
+where :math:`h_i` is the :math:`i`-th diagonal term of the matrix
   :math:`\underline{\underline{\Psi}} (\underline{\underline{\Psi}}^{\textsf{T}}\underline{\underline{\Psi}})^{-1} \underline{\underline{\Psi}}^{\textsf{T}}`
```
Comment (josephmure) [inline python/doc/theory/meta_modeling/cross_validation.rst:125]:
```suggestion
:math:`\underline{\underline{\Psi}} (\underline{\underline{\Psi}}^{\textsf{T}}\underline{\underline{\Psi}})^{-1} \underline{\underline{\Psi}}^{\textsf{T}}`
```
Maybe unindenting this line will make the build work.

**Review by josephmure — CHANGES_REQUESTED** [review]

**Review by josephmure — APPROVED** [review]
Good job.

**Comment by jschueller** [comment]
Do you know where the typos are ? The indentation changes are hiding the corrections.

**Comment by josephmure** [comment]
I am not sure any actual "typo" was corrected. The issue that this PR corrects is the faulty indentation, which resulted in a huge blank space at the top of the page: see the first point in issue #1935. IMO the PR fulfills its purpose and can be merged. You can change the commit message when you merge if you find it inappropriate.

**Comment by jschueller** [comment]
its fine then

**Pull request merged**